src/pages/position/view/index.jsx

```diff
@@ -1,32 +1,46 @@
-import React , { useEffect, useState } from "react";
+import React , { useContext, useEffect, useState } from "react";
 
-import { Position} from "@phoenixlan/phoenix.js";
+import { Position, getCurrentEvent } from "@phoenixlan/phoenix.js";
 
 import { DashboardBarElement, DashboardBarSelector, DashboardContent, DashboardHeader, DashboardSubtitle, DashboardTitle, IFrameContainer, InnerTableCell, InnerContainer, InnerContainerRow, InnerContainerTitle, InputCheckbox, InputContainer, InputDate, InputElement, InputLabel, InputText } from '../../../components/dashboard';
 
 import { useParams } from "react-router-dom";
 import { PositionMemberList } from "./memberList";
 import { PositionPermissionList } from "./permissionList";
+import { PositionDetails } from "./details";
+import { AuthenticationContext } from "../../../components/authentication";
+import { Notice } from "../../../components/containers/notice";
+import { PageLoading } from "../../../components/pageLoading";
 
 
 export const ViewPosition = (props) => {
     const { uuid } = useParams();
     const [error, setError] = useState(false);
-    const [position, setPosition] = useState([]);
+    const [currentEvent, setCurrentEvent] = useState(null);
+    const [position, setPosition] = useState(null);
+    const [usersForCurrentEvent, setUsersForCurrentEvent] = useState([])
     const [loading, setLoading] = useState(true);
     const [visibleUUID, setVisibleUUID] = useState(false);
     const [activeContent, setActiveContent] = useState(1);
 
+    // Import the following React contexts:
+    const loggedinUser = useContext(AuthenticationContext);
+
     const load = async () => {
         setLoading(true);
 
         // Get position based on UUID and return error if something fails.
         try {
+            const currentEvent = await getCurrentEvent();
             const position = await Position.getPosition(uuid);
+
+            setUsersForCurrentEvent(position.position_mappings.filter((user) => (user.event_uuid === null || user.event_uuid == currentEvent.uuid)))
```

Comment:
`user.event_uuid === null` could be `!user.event_uuid`. Since you are filtering I think its better to filter falsy values then to compare exactly.